pkg/util/oom/oom_linux.go

```diff
@@ -50,12 +49,10 @@ func getPids(cgroupName string) ([]int, error) {
 }
 
 func syscallNotExists(err error) bool {
```

Comment:
Not sure we really need this function any more. The places where you use it below have already checked if err is non-nil, and the only other check in here is for IsNotExist, which you could do down below instead of calling this.